modules/deployment/src/test/resources/ocl_test.yaml

```diff
@@ -78,32 +78,47 @@ deployment:
       validator: length(1-256)
   deployer: |
     variable "secgroup_id" {}
-
     data "huaweicloud_availability_zones" "myaz" {}
-
     data "huaweicloud_compute_flavors" "myflavor" {
-      availability_zone = data.huaweicloud_availability_zones.myaz.names[0]
-      performance_type  = "normal"
-      cpu_core_count    = 2
-      memory_size       = 4
+        availability_zone = data.huaweicloud_availability_zones.myaz.names[0]
+        performance_type  = "normal"
+        cpu_core_count    = 2
+        memory_size       = 4
     }
-
     data "huaweicloud_vpc_subnet" "mynet" {
-      name = "subnet-default"
+        name = "xpanse_subnet1"
     }
-
-    data "huaweicloud_images_image" "myimage" {
-      name        = "Ubuntu 18.04 server 64bit"
-      most_recent = true
-    }
-
     resource "huaweicloud_compute_instance" "basic" {
-      name               = "basic"
-      image_id           = data.huaweicloud_images_image.myimage.id
-      flavor_id          = data.huaweicloud_compute_flavors.myflavor.ids[0]
-      availability_zone  = data.huaweicloud_availability_zones.myaz.names[0]
-
-      network {
-        uuid = data.huaweicloud_vpc_subnet.mynet.id
-      }
+        name               = "basic"
+        image_id           = "a8601887-81d5-4eed-9338-382cf5b6d80b"
+        flavor_id          = data.huaweicloud_compute_flavors.myflavor.ids[0]
+        availability_zone  = data.huaweicloud_availability_zones.myaz.names[0]
+        network {
+            uuid = data.huaweicloud_vpc_subnet.mynet.id
+        }
+    }
+    resource "huaweicloud_evs_volume" "myvol" {
+        name              = "volume"
+        availability_zone = data.huaweicloud_availability_zones.myaz.names[0]
+        volume_type       = "SAS"
+        size              = 10
+    }
+    resource "huaweicloud_compute_volume_attach" "attached" {
+        instance_id = huaweicloud_compute_instance.basic.id

```

Comment:
In order to save resources and time, We can simplify  the deployer in the ocl_test.yaml

Reply:
I try to be as concise as possible